src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -119,8 +119,15 @@ public static boolean RequestHTTPS(
 
       if (fileStr.charAt(0) != '[' && fileStr.charAt(0) != '{') {
         // json has odd characters at the front, trim them to make valid json
-        int i = fileStr.indexOf("{");
-        fileStr = fileStr.substring(i);
+        int jsonStartIndex = fileStr.indexOf("{");
+
+        int notFound  = -1;

```

Comment:
extra space between `notFound` and `=`

If you are using IntelliJ for development, you should install the google-java-format plugin from https://plugins.jetbrains.com/plugin/8527-google-java-format and enable that for the project (once the plugin is installed, it adds a Google Java Format tab to the IntelliJ settings/preferences window). The plugin will run the google-java-format application when you run format code in IntelliJ (CTRL+ALT+L).

Reply:
Formatting was missed on one commit, this is fixed.